File: CS/Pivot_Chart_1/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace Pivot_Chart {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'gspDataSet.GSP' table. You can move, or remove it, as needed.
            this.gSPTableAdapter.Fill(this.gspDataSet.GSP);
        }

        // Adjust the X-axis labels.
        private void chartControl1_CustomizeXAxisLabels(object sender, CustomizeXAxisLabelsEventArgs e) {
            e.Staggered = false;
        }

        // Change the series view type to Line.
        private void chartControl1_BoundDataChanged(object sender, EventArgs e) {
            ((ChartControl)sender).SeriesTemplate.ChangeView(ViewType.Line);
        }

    }
}

File: VB/Pivot_Chart_1/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace Pivot_Chart
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
			' TODO: This line of code loads data into the 'gspDataSet.GSP' table. You can move, or remove it, as needed.
			Me.gSPTableAdapter.Fill(Me.gspDataSet.GSP)
		End Sub

		' Adjust the X-axis labels.
		Private Sub chartControl1_CustomizeXAxisLabels(ByVal sender As Object, ByVal e As CustomizeXAxisLabelsEventArgs) Handles chartControl1.CustomizeXAxisLabels
			e.Staggered = False
		End Sub

		' Change the series view type to Line.
		Private Sub chartControl1_BoundDataChanged(ByVal sender As Object, ByVal e As EventArgs) Handles chartControl1.BoundDataChanged
			DirectCast(sender, ChartControl).SeriesTemplate.ChangeView(ViewType.Line)
		End Sub

	End Class
End Namespace
